salt/metalk8s/orchestrate/control_plane.sls

```diff
@@ -1,8 +1,13 @@
-{%- set control_plane_ips = salt.saltutil.runner('mine.get', tgt='*', fun='control_plane_ips') %}
+{%- if pillar['bootstrap_id'] %}
+{%-   set control_plane_ips = salt.saltutil.runner('mine.get', tgt=pillar['bootstrap_id'], fun='control_plane_ip') %}
+{%- else %}
+{%-   set control_plane_ips = {} %}
+{%- endif %}
+
 {%- if pillar['bootstrap_id'] in control_plane_ips.keys() and control_plane_ips[pillar['bootstrap_id']] %}
```

Comment:
Maybe a single ternary could be more readable than these conditions (not your code, but since you're changing things around it...):

```jinja
{%- set control_plane_ip = control_plane_ips.get(pillar['bootstrap_id']) | default('localhost', true) %}
```

Reply:
Done